Front-end/glaucoma-diagnosis-system/src/router/index.ts

```diff
@@ -24,8 +25,12 @@ const routes = [
         path: '/reception',
         name: 'Reception',
         component: Reception,
+    },
+    {
+        path: '/lab',
+        nam: 'Labratory',
+        component: Labratory,
     }
-
```

Comment:
I think there are conflicts in this file. you have to resolve them.  other than that it lgtm.